pyrenew/latent/infectionswithfeedback.py

```diff
@@ -160,19 +167,23 @@ def sample(
                 **kwargs,
             )
         )
+
+        if inf_feedback_strength.ndim == Rt.ndim - 1:
+            inf_feedback_strength = jnp.expand_dims(inf_feedback_strength, 0)
```

Comment:
I think this needs to be modified based on @dylanhmorris's [comment](https://github.com/CDCgov/PyRenew/pull/440#discussion_r1759557819)

Reply:
replaced `jnp.expand_dims` with `jnp.newaxis` in https://github.com/CDCgov/PyRenew/pull/440/commits/bafe3b04021d389fe626e972297228cc586e99ce